src/expressions.rst

```diff
@@ -2101,8 +2058,8 @@ The :t:`evaluation` of a :t:`basic assignment` proceeds as follows:
    uninitialized :t:`field` of a :t:`variable`.
 
 #. :dp:`fls_hc01gtvlxba`
-   The :t:`value` of the :t:`value operand` is passed :t:`by value` into the
-   place of the :t:`assignee operand`.
+   The :t:`value` of the :t:`value operand` is :t:`passed` into the
```

Comment:
A suggestion: change all the :t:`passed` to :t:`passed <passing convention>` since I noticed that some term references are inconsistent (passed vs [pass]ed), and it makes sense to link directly to passing conventions.